acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java

```diff
@@ -442,6 +516,81 @@ private void runFullBuildFlow(BitbucketScmWorkflowJob workflowJob) throws IOExce
                 assertThat(status, successfulBuildWithKey(getBuildKey(workflowJob))));
     }
 
+    private void runFullFlowMultiBranch(boolean triggerOnRefChange, boolean triggerOnPullRequest) throws IOException, GitAPIException, InterruptedException {
```

Comment:
This is a big win for us to have proper trigger coverage in the func tests. Thanks for doing it :)